src/main/include/log4cxx/private/aprserversocket.h

```diff
@@ -31,7 +31,7 @@ namespace helpers
 class LOG4CXX_EXPORT APRServerSocket : public helpers::ServerSocket
 {
         public:
-            APRServerSocket(int port);
+            APRServerSocket(int port, bool reuseAddress = true);
```

Comment:
This causes an ABI breakage, you need to add a new constructor that takes in two parameters